### DISCUSSION TITLE
Remove list of options

### DISCUSSION THREAD

**Discussion by qsoe** [valinet/ExplorerPatcher#1595] [Q&A]
Since the re-installation of the Windows and ExplorerPatcher on my laptop, a strange list appeared on my taskbar. I tried to search for it, but I have no idea how it is called. It is a list of shortcuts next to my tray icons, with the word "Desktop" above them, and a double arrow below, which opens up the full list, see below.

It is not horrible on the big monitor, but it takes up a lot of useful space when I work on my laptop alone. I spent about 20 minutes in ExplorerPatcher properties, still I cannot turn it off. 
I want to get rid of everything in the maroon box. How can I do it?

windows 11: 22621.1848
EP: 22621.1555.55.1

![desktop-both](https://github.com/valinet/ExplorerPatcher/assets/23166556/79a383ab-2c02-4bc1-adf2-8bb8b154d6c6)

**Comment by vippium** [comment]
Just turn it off by R-clicking on taskbar, then, in toolbars,

**Reply by qsoe** [reply]
This seems incomplete

**Reply by vippium** [reply]
Can't you share a video... Screenshot is not sufficient for understanding it completely. and also can you take ss while the taskbar on bottom, as I only use bottom. So I can have better understading with the problem

**Reply by qsoe** [reply]
Yes, it was the "desktop" toolbar. Thanks
